vaadin-testbench-core/src/main/java/com/vaadin/testbench/TestBenchTestCase.java

```diff
@@ -33,6 +34,10 @@
 public abstract class TestBenchTestCase
         implements HasDriver, HasTestBenchCommandExecutor, HasElementQuery {
 
+    static {
+        UsageStatistics.markAsUsed("testbench/TestBenchTestCase", null);
```

Comment:
This should probably use TestBench version instead of null. There is a helper somewhere in the project